community/kernel/src/main/java/org/neo4j/kernel/impl/store/AbstractDynamicStore.java

```diff
@@ -254,11 +255,21 @@ public int getNumberOfReservedLowIds()
     @Override
     protected void readAndVerifyBlockSize() throws IOException
```

Comment:
The point of this method is that it is implemented differently in `AbstractStore`, AbstractDynamicStore` and `RelationshipGroupStore`.